frontend/src/components/header.js

```diff
@@ -1,33 +1,122 @@
-    />
-  </header>
-);
+import hmccLogo from "../images/hmcc-ripple-white.svg";
+import mapPinLogo from "../images/icons/map-pin.svg";
+import dropDown from "../images/icons/dropdown.svg";
+import x from "../images/icons/X.svg";
+
+const Header = () => {
+  const browseList = [
+    { title: "About", route: "/about" },
+    { title: "Connect", route: "/connect" },
+    { title: "Next Steps", route: "/next-steps" },
+    { title: "Events", route: "/events" },
+    { title: "Watch", route: "/watch" },
+    { title: "Give", route: "/give" },
+  ];
+
+  const locationsList = [
+    { title: "Ann Arbor", route: "/" },
+    { title: "Austin", route: "https://austin.hmcc.net/" },
+    { title: "Detroit", route: "https://detroit.hmcc.net/" },
+    { title: "Hong Kong", route: "https://hongkong.hmcc.net/" },
+    { title: "Jakarta", route: "https://jakarta.hmcc.net/" },
+  ];
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const path = window.location.pathname;
+  const containerStyle = "flex flex-row h-[60px] [@media(min-width:900px)]:h-[100px] flext-start bg-Primary-700 items-center justify-center pl-[18px] xl:pl-[80.286px] [@media(min-width:900px)]:pt-[40px] [@media(min-width:900px)]:pb-[17px] pr-[16px] xl:pr-[65px] lg:gap-[20px] xl:gap-[140px] [@media(min-width:1440px)]:gap-[220px] position-fixed"
+  const logoStyle = "w-[80px] h-[32px] mb-0";
+  const dropDownStyle = "w-[40px] h-[40px] mb-0"
+  const textStyle = "text-Shades-0 text-base leading-normal no-underline";
+  const mapPinStyle = "mb-0 shrink-0 h-[20px] w-[15.7px] peer hover:bg-[#1A56D6]";
+  const borderStyle = "flex items-center py-2 px-4 md:px-6 rounded-lg border-2 border-[#FF8069] shadow-md";
+
+  // var menu = document.getElementById("menu");
+
+  // this function is used to open the menu
+  // function openMenu() {
+  //     menu.classList.remove("w-0", "h-0", "opacity-0");
+  //     menu.classList.add("w-screen", "h-screen", "opacity-95");
+  // }
+
+  // this function is used to close the menu
+  // function closeMenu() {
+  //     menu.classList.remove("w-screen", "h-screen", "opacity-95");
+  //     menu.classList.add("w-0", "h-0", "opacity-0");
+  // }
+
+
+  return (
+    <header>
+      <div className={`${containerStyle}`}>
+        <div className="flex flex-row w-full justify-between [@media(min-width:900px)]:hidden relative">
+          <a href="/">
+            <img alt="hmcc logo" className={logoStyle} src={hmccLogo} />
+          </a>
```

Comment:
```suggestion
          <Link to="/">
            <img alt="hmcc logo" className={logoStyle} src={hmccLogo} />
          </Link>
```
Could you use the gatsby Link Component?  this is what been using for all our links.